### PR TITLE
feat(financial-templates-lib): Add various altcoin price feed configs

### PR DESCRIPTION
**Pull request by smb2796**
**Motivation**

To support the AAVEUSD, USDAAVE, UMAUSD, USDUMA, SNXUSD, USDSNX, LINKUSD, USDLINK, UNIUSD and USDUNI price identifiers added in [UMIP-57](https://github.com/UMAprotocol/UMIPs/blob/master/UMIPs/umip-57.md), default price feed configs need to be added.

All of these price identifiers use Coinbase Pro, Binance and OKEx and are all available on Cryptowatch. 

**Summary**

Added medianizer configs for each price identifier listed.

**Testing**

Check a box to describe how you tested these changes and list the steps for reviewers to test.

- [ ]  Ran end-to-end test, running the code as in production
- [ ]  New unit tests created
- [ ]  Existing tests adequate, no new tests required
- [X]  All existing tests pass
- [ ]  Untested

Ran USD/XYZ price feeds in a monitor bot locally.
 
**Issue(s)**

NA

### CODE REVIEW

File: packages/financial-templates-lib/src/price-feed/DefaultPriceFeedConfigs.js [hunk 1]
```diff
@@ -563,6 +563,101 @@ const defaultConfigs = {
     uniswapAddress: "0xedf187890af846bd59f560827ebd2091c49b75df",
     twapLength: 7200,
     invertPrice: true
+  },
+  USDAAVE: {
```
Comment (nicholaspai) [inline packages/financial-templates-lib/src/price-feed/DefaultPriceFeedConfigs.js:567]: Think you're going to need a `lookback` on any pricefeed that uses a Cryptowatch pricefeed.

Can set it under `invertPrice`: `lookback: 7200`, similar to `USDETH`

**Review by nicholaspai — COMMENTED** [review]
Looks great but just check my 1 comment

**Review by nicholaspai — APPROVED** [review]
Actually disregard my comment, these look good. USDETH doesn't have a lookback!

**Comment by smb2796** [comment]
Sweet! Thanks Nick

**Pull request merged**